shared/src/store/action-reducers/emergency-operation-center.ts

```diff
@@ -15,6 +39,25 @@ export class AddLogEntryAction implements Action {
     public readonly message!: string;
 }
 
+export class SendAlarmGroupAction implements Action {
+    @IsValue('[Emergency Operation Center] Send Alarm Group' as const)
+    public readonly type = '[Emergency Operation Center] Send Alarm Group';
+    @IsString()
+    @MaxLength(255)
+    public readonly name!: string;
```

Comment:
Please call this one `trainerName` or `clientName` (as in the `EocLogEntry` class). I first though it was the alarm group's name...

Reply:
Done in 9fae3e2535a1c52601085fc6126acc7d4efa034b